crates/types/src/vote.rs

```diff
@@ -180,3 +182,20 @@ impl<TYPES: NodeType, VOTE: Vote<TYPES>, CERT: Certificate<TYPES, Voteable = VOT
 
 /// Mapping of commitments to vote tokens by key.
 type VoteMap2<COMMITMENT, PK, SIG> = HashMap<COMMITMENT, (U256, BTreeMap<PK, (SIG, COMMITMENT)>)>;
+
+/// Payload for the `HotShotEvents::VoteNow` event type. The proposal and leaf are
+/// obtained via a `QuorumProposalValidated` event being processed.
+#[derive(Eq, Hash, PartialEq, Debug, Clone)]
+pub struct VoteDependencyData<TYPES: NodeType> {
```

Comment:
Is this used anywhere or just a placeholder?

Reply:
It's used in the event type and I also use it in the unit test (pushing in a sec)